Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -123,32 +123,51 @@ def binaryProductLimit (X Y : SSet) : IsLimit (binaryProductCone X Y) where
     simp only [← ht ⟨WalkingPair.right⟩, ← ht ⟨WalkingPair.left⟩, binaryProductLift]
     congr
 
-/-- `Prod X Y` is a binary product for `X` and `Y`. -/
+/-- `prod X Y` is a binary product for `X` and `Y`. -/
 def binaryProductLimitCone (X Y : SSet) : Limits.LimitCone (pair X Y) :=
   ⟨_, binaryProductLimit X Y⟩
 
-/-- The categorical binary product of simplicial sets is `Prod X Y`. -/
-noncomputable def binaryProductIso (X Y : SSet) : X ⨯ Y ≅ Prod X Y :=
+/-- The categorical binary product of simplicial sets is `prod X Y`. -/
+noncomputable def binaryProductIso (X Y : SSet) : X ⨯ Y ≅ prod X Y :=
   limit.isoLimitCone (binaryProductLimitCone X Y)
 
 @[simp]
 lemma binaryProductIso_hom_comp_fst (X Y : SSet) :
-    (binaryProductIso X Y).hom ≫ Prod.fst = Limits.prod.fst := by aesop
+    (binaryProductIso X Y).hom ≫ prod.fst = Limits.prod.fst := by aesop
 
 @[simp]
 lemma binaryProductIso_hom_comp_snd (X Y : SSet) :
-    (binaryProductIso X Y).hom ≫ Prod.snd = Limits.prod.snd := by aesop
+    (binaryProductIso X Y).hom ≫ prod.snd = Limits.prod.snd := by aesop
 
 @[simp]
 lemma binaryProductIso_inv_comp_fst (X Y : SSet) :
-    (binaryProductIso X Y).inv ≫ Limits.prod.fst = Prod.fst := by
+    (binaryProductIso X Y).inv ≫ Limits.prod.fst = prod.fst := by
   simp [binaryProductIso, binaryProductLimitCone]
 
 @[simp]
 lemma binaryProductIso_inv_comp_snd (X Y : SSet) :
-    (binaryProductIso X Y).inv ≫ Limits.prod.snd = Prod.snd := by
+    (binaryProductIso X Y).inv ≫ Limits.prod.snd = prod.snd := by
   simp [binaryProductIso, binaryProductLimitCone]
 
+/-- Constructing an n-simplex in X ⨯ Y from an n-simplex in X and an n-simplex in Y. -/
+@[simp]
+noncomputable
+def prodMk {X Y : SSet} {n : SimplexCategoryᵒᵖ} (x : X.obj n) (y : Y.obj n) :
+    (X ⨯ Y).obj n := ((binaryProductIso X Y).inv).app n ⟨x, y⟩
```

Comment:
My idea was that the following statement should be a simp lemma:
```lean
lemma prodMk_fst {X Y : SSet} {n : SimplexCategoryᵒᵖ} (x : X.obj n) (y : Y.obj n) :
    (Limits.prod.fst : X ⨯ Y ⟶ X).app _ (prodMk x y) = x
```
In order to achieve this, I think we need `_apply` versions of some lemmas, like `binaryProductIso_inv_comp_fst_apply`. But then, I would suggest not tagging `prodMk` with the simp attribute, because I would argue that when we need to work with the categorical product, we may not want to make the other model of product `SSet.prod` appear explicitly.

Reply:
Thanks, I'm still figuring out when the `simp` tag is sensible. I'll remove it from `prodMk`. I also finished `prodMk_fst` and `prodMk_snd` earlier today, and I'm going to push them with some other lemmas tomorrow.